src/test/run-pass/consts/allow_const_fn_ptr.rs

```diff
@@ -0,0 +1,16 @@
+#![feature(rustc_attrs, staged_api)]
```

Comment:
Nit: move this to `ui/consts/min_const_fn/allow_const_fn_ptr2.rs` and add `// run-pass` as the first line since we want to get rid of the `run-pass` folder one day afaik (and its neater if they are colocated)

Reply:
done.